src/Microsoft.Diagnostics.EventFlow.Inputs.DiagnosticSource/Configuration/DiagnosticSourceConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.Diagnostics.EventFlow.Configuration
```

Comment:
In the main EventFlow repo all source files need to have Microsoft Copyright notice--see existing code